src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -44,20 +40,18 @@ import { buildPatternsScene } from './Breakdowns/PatternsBreakdownScene';
 import { GoToExploreButton } from './GoToExploreButton';
 import { buildLogsListScene } from './LogsListScene';
 import { testIds } from 'services/testIds';
-import { PageScene } from './PageScene';
 import { sortLabelsByCardinality } from 'services/filters';
 import { SERVICE_NAME } from 'Components/ServiceSelectionScene/ServiceSelectionScene';
+import { getSlug } from '../Pages';
 
 export interface LokiPattern {
   pattern: string;
   samples: Array<[number, string]>;
 }
 
-export type ActionViewType = 'logs' | 'labels' | 'patterns' | 'fields';
-
-interface ActionViewDefinition {
+interface BreakdownViewDefinition {
   displayName: string;
-  value: ActionViewType;
+  value: SLUGS;
```

Comment:
Related with my previous comment; at first glance looks like some enum or constant, not a type.

Reply:
SLUGS is an enum

Reply:
Renamed tho